src/main/java/heronarts/lx/parameter/LXListenableNormalizedParameter.java

```diff
@@ -57,4 +58,31 @@ public boolean isMappable() {
     return this.mappable;
   }
 
+  public LXListenableNormalizedParameter setWrappable(boolean wrappable) {
+    this.wrappable = wrappable;
+    return this;
+  }
+
+  @Override
+  public boolean isWrappable() {
+    return this.wrappable;
+  }
+
+  public LXListenableNormalizedParameter incrementNormalized(double amount) {
+    return incrementNormalized(amount, this.wrappable);
```

Comment:
Should use `isWrappable()` here in case a subclass were to override it... probably a non-issue, I'll quickly fix post-merge.